attest/core/src/types/target_info.rs

```diff
@@ -74,7 +72,7 @@ impl TargetInfo {
 
     /// Retrieve the target enclave's measurement
     pub fn mr_enclave(&self) -> MrEnclave {
-        self.0.mr_enclave.into()
+        MrEnclave::from(self.0.mr_enclave.m)
```

Comment:
:upside_down_face: I'm not sure it matters much if one uses `into()` vs `from()`, just noticed the diff that I don't think is necessary.
```suggestion
        self.0.mr_enclave.into()
```

I generally lean on using `into()` when I can get away with it, and use the `from()` version when the compiler can't determine the type, but I haven't really looked to see if there is a rustic convention.

Reply:
:facepalm: nvm it fails to build this way